packages/psss-server/src/app/addRoutesToApp.ts

```diff
@@ -59,14 +61,20 @@ export function addRoutesToApp(app: Express) {
    */
   app.post('/v1/login', handleWith(LoginRoute));
   app.post('/v1/logout', handleWith(LogoutRoute));
+  app.post(
+    '/v1/confirmedWeeklyReport/:organisationUnitCode',
+    handleWith(SubmitConfirmedCountryWeeklyReportRoute),
+  );
```

Comment:
I decided to move this endpoint to `POST`. Given that we don't actually send any data for that endpoint, and instead we are telling it to 'take the unconfirmed data and submit it as confirmed data' it feels more _POSTy_ to me?

An argument could be made to rename the class as well but I don't feel too strongly about it...

Reply:
Re the `POST` move - yep that makes sense.

I actually _do_ feel strongly about the class name! I don't mind what we call each of them, but I feel there's already confusion about the different behaviours of the two "Submit" endpoints. One responds to the "Save" button, the other to the "Confirm" button. In the former case, the user knows they're editing data. In the latter, they feel more like they're just flagging data as confirmed - the fact that it's also upserting a survey response is an implementation detail within our system. Perhaps we can reflect that with a difference in the naming of the two handlers? 

And in writing that out I've talked myself around to realising that the handler names probably _should_ relate more to our internal implementation details. So now I don't feel too strongly either! But I would still lean towards naming this one here as "`ConfirmCountryWeeklyReportRoute`".

Reply:
I did actually think about renaming it to `ConfirmCountryWeeklyReportRoute` as well, however that's dangerously close to our existing `ConfirmedCountryWeeklyReportRoute` 🙀 

I think actually that we need a full rename on the routes to be:
`FetchCountryWeeklyReportRoute = GET /weeklyReport/:orgUnitCode` 
`FetchConfirmedCountryWeeklyReportRoute = GET /confirmedWeeklyReport/:orgUnitCode`
`FetchConfirmedWeeklyReportRoute = GET /confirmedWeeklyReport`

`SubmitCountryWeeklyReportRoute = PUT /weeklyReport/:orgUnitCode`
`DeleteCountryWeeklyReportRoute = DELETE /weeklyReport/:orgUnitCode`

`ConfirmCountryWeeklyReportRoute = POST /confirmedWeeklyReport/:orgUnitCode or POST /weeklyReport/:orgUnitCode` thoughts on this?